projects/ngx-prx-styleguide/src/lib/upload/file/index.ts

```diff
@@ -1,3 +0,0 @@
-export * from './duration.pipe';
```

Comment:
Are these exports not used anywhere?

Ah, I see you removed a bunch of the barrel files.  So maybe they weren't necessary.

Reply:
Ah... and reading the rest of the PR, I see these were causing problems.  +100!